### PR TITLE
Portfolio mobile version (2nd milestone)

### PR DESCRIPTION
**Pull request by saqibjavaiddev**
For the second milestone in building my portfolio website, I created the mobile website section where I listed my portfolio projects. I build the work "Work" section and the "About myself" section. The learning objectives for this project are: 

- Understand how to parse a Figma design to create a UI.
- Use Flexbox to place elements in the page.
- Use CSS Grid to place elements in the page.
- Build a personal portfolio site.

### CODE REVIEW

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that some linters check are not successful. Keep in mind that Linter is there in order to help you to have a good code base quality. 

linter

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by saqibjavaiddev** [comment]
Sorry I forgot about linters check, tried to undo submission but its was claimed

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @saqibjavaiddev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
- Please consider this review a chance to learn and improve your coding skills, because that is what you are here for.

## Highlights
- Good HTML structure
- Usage of semantic HTML tags.
- Good CSS.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Since you asked for the guide on how to fix linters in vs code, I suggest following this [link](https://github.com/microverseinc/linters-config/tree/master/html-css).

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -18,20 +18,210 @@
...
+            <a href=""><i class="fa fa-github"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fa fa-angellist"></i></a>
+          </li>
+        </ul>
+      </section>
+      <section class="work">
+        <div class="project">
+          <img src="assets/img/work1.png" alt="Image project" class="project_image"/>
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:50]: [muneebulrehman](https://github.com/muneebulrehman) Image width fixed

File: index.html [hunk 2]
```diff
@@ -18,20 +18,210 @@
...
+            <footer>
+              <ul class="skills">
+                <li>React</li>
+                <li>Redux</li>
+                <li>API REST</li>
+                <li>Javascript</li>
+                <li>JEST</li>
+                <li>CSS3</li>
+                <li>html5</li>
+              </ul>
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:68]: Margin added

File: index.html [hunk 3]
```diff
@@ -18,20 +18,210 @@
...
+                    <img class="hide"  src="assets/img/select.png" alt="selected" >
+                  </div>
+                </div>
+                <div class="frams">
+                </div>
+              </div>
+                  <hr/>
+              </div>
+            </div>   
+        </section>
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:224]: Margin bottom added

File: index.html [hunk 4]
```diff
@@ -18,20 +18,210 @@
...
+                <li>JEST</li>
+                <li>CSS3</li>
+                <li>html5</li>
+              </ul>
+              <div class="external_link"><a href="#">See Project</a></div>
+            </footer>
+          </div>
+        </div>
+      </section>
+        <section id="about_me">
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:146]: border radius added

File: index.html [hunk 5]
```diff
@@ -18,20 +18,210 @@
...
+                    <a href=""><i class="fa fa-github"></i></a>
+                  </li>
+                  <li>
+                    <a href=""><i class="fa fa-angellist"></i></a>
+                  </li>
+                </ul>
+                <a id="my_resume" target="_blank" href="#">Get my resume</a>
+              </div>
+            </div>
+            <div id="my_skills">
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:175]: Added in lis form

File: index.html [hunk 6]
```diff
@@ -18,20 +18,210 @@
...
+              <div id="languages_skills">
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:176]: Done added

File: index.html [hunk 7]
```diff
@@ -18,20 +18,210 @@
...
+                     <li class="lang">
+                       <img src="assets/img/css.png" alt="css image">
+                       <p>CSS</p>
+                     </li>
+                </ul>
+              </div>
+              <div>
+                <div> <hr/></div>
+              </div>
+              <div id="frameworks">
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:201]: Added

File: index.html [hunk 8]
```diff
@@ -18,20 +18,210 @@
...
+                <div class="lisframes">
+                  <p>Frameworks</p>
+                  <div class="show-frameworks">
+                    <img class="hide"  src="assets/img/select.png" alt="selected" >
+                  </div>
+                </div>
+                <div class="frams">
+               </div>
+              <hr/>
+              <div id="frameworks1">
```
[the remark this answers is not in the document]
Comment (saqibjavaiddev) [reply index.html:207]: Added

**Review by muneebulrehman — APPROVED** [review]
Hi @saqibjavaiddev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳 
Congratulations! 🎉 

# Status: Approved

## Highlights
- Good documentation
- Linter checks are passing
- Amazing job following up with the previously raised issues.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by saqibjavaiddev** [comment]
Hi [rindrajosia](https://github.com/rindrajosia) and [uneebulrehman](https://github.com/muneebulrehman) Please help me to setup linters on my vscode so I don't need to commit again and again to check linter errors or some tip and trick to fix on vscode directly

**Pull request merged**